tools/logging.py

```diff
@@ -27,22 +41,13 @@ def error(msg, rc=1):
 def log(msg):
     """
     Log message
+
+    Args:
+        msg (string): error message to be printed
+
+    Returns:
+        does not return anything
```

Comment:
Well, in fact, `None` is always implicitly returned... Not sure how the rephrase this

Reply:
Replaced it with

```
        None (implicitly)
```